### DISCUSSION TITLE
Sepolia BasicNft conctract deployed successfully, but strange terminal feedback.

### DISCUSSION THREAD

**Discussion by adamwestlake** [Cyfrin/foundry-full-course-cu#1603] [Q&A]
Hello cyfrin enjoyers,

I am steaming through the NFT course, I found that my contract successfully deployed, but in the terminal, forge is informing that it could find the contract deployed to 0x23...

I can see via ethscan sepolia it is deployed, but I am wondering to whos address?

Thanks, Adam

![help1](https://github.com/Cyfrin/foundry-full-course-f23/assets/103932610/f4e502a4-f8e6-4882-9727-2a0962d1115b)
![help2](https://github.com/Cyfrin/foundry-full-course-f23/assets/103932610/72bc395e-c8b2-422e-bb6a-cb3914ddf53e)

**Comment by EngrPips** [comment]
Hello @adamwestlake , I am convinced it didn't say `it couldn't find your contract`. but instead it said it didn't find any contract that match exactly your contract bytecode. I mean you see that it is successfully deployed.

Mostly that message is something related to verification on etherscan.
